CMakeLists.txt

```diff
@@ -1,10 +1,54 @@
-cmake_minimum_required(VERSION 3.20.0 FATAL_ERROR) # Maybe we can reduce this?
+cmake_minimum_required(VERSION 3.0)
 
 project(
     VulkanMemoryAllocator-Hpp
     VERSION 0
     DESCRIPTION "C++ bindings for the VulkanMemoryAllocator library."
     LANGUAGES C CXX)
 
+find_package(Vulkan REQUIRED)
+find_package(VulkanMemoryAllocator REQUIRED)
```

Comment:
I have little knowledge of CMake, but I believe this will not find VMA if it was checked out from the submodule rather than being installed in the system, will it? Maybe we can support such case too?

Reply:
Fixed

Reply:
I meant to support both scenarios, something like if `VulkanMemoryAllocator/CMakeLists.txt` exists, then `add_subdirectory`, otherwise `find_package`. Can you add something like this, please?

Reply:
Do you think that would be best? After your suggestion I realized that since you have VMA bundled anyway, maybe it makes more sense to simply use the version packaged with it. Do you just want it to work whether the submodule is checked out or not? I think that would be a valid reason to include `find_package`

Reply:
Some users may want to use their own VMA instead of bundled one, there's even a [pre-include guard](https://github.com/YaaZ/VulkanMemoryAllocator-Hpp/blob/master/include/vk_mem_alloc.hpp#L4) in `vk_mem_alloc.hpp` so that user can include VMA from custom location before including VMA-Hpp. So I think it's a valid case when user doesn't need to check out the submodule at all. Here's the related discussion: https://github.com/YaaZ/VulkanMemoryAllocator-Hpp/issues/6